model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/v2/tp_model.py

```diff
@@ -151,6 +152,14 @@ def generate_tp_model(default_config: OpQuantizationConfig,
     # this configuration will be used for the operation quantization:
     default_configuration_options = tp.QuantizationConfigOptions([default_config])
 
+    # const configuration is similar to the default eight bit configuration except for PoT
+    # quantization method for the constant.
```

Comment:
I think we should elaborate more here, something like:
` # Create a QuantizationConfigOptions for quantizing constants in functional ops.
... (what you wrote)....
Since the constants are not named attributes of the layer, we use the default_weight_attr_config to define the desired quantization properties for them.
`